dist/test_data.py

```diff
@@ -41,11 +41,18 @@ def __gt__(self, other):
 
     def __ge__(self, other):
         return self.name >= other.name
+#
+# A generic configuration used by some components to define their tick rate.
+#
+throttle_config = [
+    Config('ops_per_second',1,r'''
+        The number of times an operation should be performed per second''', min=1,max=1000),
+]
 
 #
 # Record config specifies the format of the keys and values used in the database
 #
-record_config = [
+record_config = throttle_config + [
```

Comment:
Do you need the throttle config in the record config ?

Reply:
The `update_config` and `insert_config` in the workload generator would expect a throttle, this is my way of giving them one without using a subconfig. Perhaps we could find better names?

Reply:
I feel that throttle config should not be part of record config, I would keep them seperate.
Would it hurt to have :
subconfig=[record_config,throttle_config] ?

Reply:
We'll do this in another ticket